services/adminer/.env

```diff
@@ -1 +1,2 @@
 MONITORING_DOMAIN=monitoring.osparc.local
+POSTGRES_DEFAULT_SERVER=postgres
```

Comment:
hmm not sure you sed it in the script above. this should be equal to ${POSTGRES_HOST}